builtin/logical/pki/path_revoke.go

```diff
@@ -57,11 +61,28 @@ func (b *backend) pathRevokeWrite(ctx context.Context, req *logical.Request, dat
 	// We store and identify by lowercase colon-separated hex, but other
 	// utilities use dashes and/or uppercase, so normalize
 	serial = strings.Replace(strings.ToLower(serial), "-", ":", -1)
+	ns, err := namespace.FromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
 
 	b.revokeStorageLock.Lock()
 	defer b.revokeStorageLock.Unlock()
 
-	return revokeCert(ctx, b, req, serial, false)
+	labels := []metrics.Label{
+		metricsutil.NamespaceLabel(ns),
+	}
+	key := metricsKey(req, "revoke")
```

Comment:
Is there a reason that this can't use the metrics wrap that the other calls use?

Reply:
Yes, the metrics wrap assumes "role" is a parameter and grabs it.  revoke just operates on serial numbers.